bedrock/firefox/templates/firefox/channel/ios.html

```diff
@@ -23,27 +22,20 @@
 {% endif %}
 {% endblock %}
 
-{#- This will appear as <meta property="og:description"> which can be used for social share -#}
 {% block page_og_desc %}{{ self.page_desc() }}{% endblock %}
 
-{% block body_id %}platform-ios{% endblock %}
-
 {% block channels %}
-<section id="testflight" class="channel">
-  <div class="container">
-    <header>
-      <h2>{{_('TestFlight')}}</h2>
-      <p>{{_('Test beta versions of Firefox for iOS via Apple’s TestFlight program.')}}</p>
-    </header>
-    <a href="{{ url('firefox.ios.testflight') }}" class="button testflight-cta">{{_('Sign up now')}}</a>
```

Comment:
Looks like this `testflight-cta` class name is being relied upon by a functional test: https://github.com/mozilla/bedrock/blob/master/tests/pages/firefox/channel/ios.py#L14